src/Utils/BlockTemplateUtils.php

```diff
@@ -109,6 +109,9 @@ public static function gutenberg_build_template_result_from_post( $post ) {
 		$template->has_theme_file = $has_theme_file;
 		$template->is_custom      = true;
 		$template->post_types     = array(); // Don't appear in any Edit Post template selector dropdown.
+		if ( 'wp_template_part' === $post->post_type ) {
+			$template->area = 'uncategorized';
+		}
```

Comment:
I think we should get the template area from the database instead of hard-coding it like this. We can get the area of a template part using `get_the_terms( $post, 'wp_template_part_area' );`